### DISCUSSION TITLE
Date group by not working

### DISCUSSION THREAD

**Discussion by Usama420-git** [grafana/grafana-infinity-datasource#492] [Q&A] [answered]
Date group by not working as expected when date range is from 1-nov the group by feature working correctly but when the date range set to after 1 nov date group by not working see the screenshots below

Grafana version: 9.3.1
Data Source: infinity

![perfect time range](https://user-images.githubusercontent.com/51694624/208660943-e2c078d2-71df-4dd3-bd92-3dbd74294d35.PNG)
![wrong time range](https://user-images.githubusercontent.com/51694624/208660947-e71f33c7-d9da-4fe0-8fb1-9b03f7227e05.PNG)

**Comment by yesoreyeram** [comment, marked as answer]
Not specific to the plugin. Transformations are coming from Grafana itself.
